src/lib/core/fs.js

```diff
@@ -141,6 +141,10 @@ function existsSync() {
   return restrictPath(fs.existsSync, fs.existsSync, 1, arguments);
 }
 
+function ensureFileSync() {
+  return restrictPath(fs.ensureFileSync, fs.existsSync, 1, arguments);
```

Comment:
```suggestion
  return restrictPath(fs.ensureFileSync, fs.ensureFileSync, 1, arguments);
```
^ all the others have the first and second arguments being the same, is this one a special case?

Reply:
Updated